lib/Connection.php

```diff
@@ -520,42 +523,51 @@ private function establishConnection() {
 				}
 			}
 
-			$bindStatus = false;
-			$error = -1;
```

Comment:
the whole connect logic can be done without the tracking of bind status er error codes ...